terminus-ui/autocomplete/src/autocomplete.component.ts

```diff
@@ -534,7 +547,9 @@ export class TsAutocompleteComponent implements OnInit,
             throw Error('ngModel must be an array of values');
           }
           this.autocompleteFormControl.setValue(this.ngControl.value);
-          this.setSelections();
+          if (!this.allowMultiple) {
+            this.searchQuery = this.displayFormatter(this.ngControl.value[0]);
```

Comment:
why `searchQuery` needs to be updated only if it's single selection?

Reply:
because `searchQuery` is only really used to display a selected item in single selection only. We show chips in multiple selection instead